src/components/AppletsTable/Table/Table.types.ts

```diff
@@ -0,0 +1,49 @@
+export type Order = 'asc' | 'desc';
+
+export type Cell = {
+  align?: 'center' | 'inherit' | 'justify' | 'left' | 'right';
+  width?: string;
+};
+
+export type HeadCell = Cell & {
+  id: string;
+  label: string;
+  enableSort?: boolean;
+};
+
+export type Row = {
```

Comment:
Maybe it's better to divide properties for folder and applet in their own types and use type for example:  Row = FolderRow & AppletRow. And remove comments

Reply:
as discussed, will be changed in next MR